### PR TITLE
Feature/tasks details

### PR DESCRIPTION
**Pull request by wilsoncooked**

### CODE REVIEW

File: src/components/task/TaskDetailCard.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            <div className='col-lg-12 col-md-12 col-sm-12'>
+              <div className='card'>
+                <div className='card-header card-header-rose'>
+                  <h4 className='card-title'>Task: {this.state.task.title}</h4>
+                </div>
+                {/* summary */}
+                <div className='card-body'>
+                  <div className='row'>
+                    <div className='col-lg-4 col-md-12 col-sm-12'>
+                      <ProjectInfo projectId={this.state.task.projectId} />
```
Comment (laura-mso) [inline src/components/task/TaskDetailCard.js:68]: so nice that you added a link to the project!

**Review by jenna1k — DISMISSED** [review]
wow looks amazing!!

**Review by aliashkevich — CHANGES_REQUESTED** [review]
New Task functionality doesn't work in this PR

File: src/components/task/ProjectInfo.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+              state: {
+                id: projectId,
+              },
+            }}>
+            <button type='button' class='btn btn-success'>
+              <b>
+                {this.state.projects.find(
+                  project => project.id === this.props.projectId,
+                ).title
+                  ? this.state.projects.find(
```
Comment (laura-mso) [inline src/components/task/ProjectInfo.js:52]: like this you have to do the find operation twice. maybe you can assign it to a variable above and than use that one here twice

**Review by laura-mso — COMMENTED** [review]
The result looks really great!

File: src/components/task/ProjectInfo.js (hunk 2 above)
Comment (wilsoncooked) [reply src/components/task/ProjectInfo.js:52]: I couldn't work out how to do this inside the return :/ If you have an idea please let me know

Comment (laura-mso) [reply src/components/task/ProjectInfo.js:52]: I think you can create the variable above the return

**Review by aliashkevich — APPROVED** [review]
well done)

**Pull request merged**